packages/calcite-components/readme.md

```diff
@@ -44,6 +44,8 @@ Choose one of the two builds provided by Calcite Components.
 
 #### Custom Elements
 
+<!-- TODO: Get suggestions for what this should be replaced with -->
```

Comment:
idea: replace the Custom Elements and Distribution sections with a link to https://developers.arcgis.com/calcite-design-system/get-started/?

That way there is one less place to keep up to date